mocha.opts

```diff
@@ -0,0 +1 @@
+--timeout 10000
```

Comment:
There is already a `.mocharc.yml` file in the repo (`mocha.opts`). Let's move that there!

Reply:
something kept generating this file but I'll remove it and use the rc